Example/Swift Example/View Controllers/UserAuthenticationViewController.swift

```diff
@@ -204,6 +214,61 @@ class UserAuthenticationViewController: UITableViewController {
         }
     }
 
+    func requestVerification() {
+        guard let user = SKYContainer.default().auth.currentUser else {
+            return
+        }
+
+        let alert = UIAlertController(title: "Request Verification", message: "Select the field for verification. This example only supports email and phone.", preferredStyle: .alert)
+
+        let verifyableRecordKeys = ["email", "phone"]
+        for recordKey in verifyableRecordKeys {
+            let enabled = (user[recordKey] as? String)?.isEmpty == false
+            let title = enabled ? recordKey : "\(recordKey) (missing)"
+            let action = UIAlertAction(title: title, style: .default, handler: { (_) in
+                SKYContainer.default().auth.requestVerification(recordKey, completion: { (error) in
+                    if let error = error {
+                        self.showError(error, completion: nil)
+                        return
+                    }
+                })
+            })
+            action.isEnabled = enabled
+            alert.addAction(action)
+        }
+        alert.addAction(UIAlertAction(title: "Cancel", style: .default, handler: nil))
+        alert.preferredAction = alert.actions.last
+        self.present(alert, animated: true, completion: nil)
+    }
+
+    func enterVerificationCode() {
+        guard SKYContainer.default().auth.currentUser != nil else {
+            return
+        }
+
+        let alert = UIAlertController(title: "Enter Verification Code", message: nil, preferredStyle: .alert)
+        alert.addTextField { (textField) in
+            textField.placeholder = "Code"
+        }
+        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: { (_) in
+            let code = alert.textFields?.first?.text ?? ""
+
+            if code == "" {
+                return
+            }
+
+            SKYContainer.default().auth.verifyUser(withCode: code, completion: { (user, error) in
+                if let error = error {
+                    self.showError(error, completion: nil)
```

Comment:
After discussion, we will update the error handling to check code and show incorrect error code message.

Reply:
Should we show user verified status in list, so that user know verification success?

Reply:
> Should we show user verified status in list, so that user know verification success?

Of course!